### DISCUSSION TITLE
Previous Isaac Orbit version API documentation

### DISCUSSION THREAD

**Discussion by nikolaradulov** [isaac-sim/IsaacLab#1146] [Q&A]
### Proposal

Make the API documentation for the old Isaac Orbit version (the one build on Isaac 2022.2.0 ) available

### Motivation

The new version of Orbit is too high level for creating the environments I need, as such I would like to keep developing on the old version of orbit. . It's likely that other users share similar concerns, emphasizing the necessity of retaining documentation for the old API.

### Checklist

- [x] I have checked that there is no similar issue in the repo (**required**)

**Comment by Mayankm96** [comment]
Hi @nikolaradulov 

It doesn't make sense to use/support the previous version of Orbit since it depends on 2022.2, which is very outdated. You can just make your environment class based on the previous version or inherit from `RLTaskEnv` and override the functions you need to modify. We have some users doing something like this. Soon we will add examples on our side to also just showcase this workflow.

But the general comment on having release-based docs, I agree we should do that :)

PS: If you run `./orbit.sh -d` you should be able to build the docs locally if you need them for immediate reference.
